keyboards/mlego/m60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,152 @@
+    _______, _______, _______, _______, _______, _______, _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+),
+
+[_ADJ] =  LAYOUT_ortho_5x12(
+    _______, RESET,   _______, _______, _______, _______, RGB_TOG, RGB_MOD, RGB_RMOD,RGB_M_G, RESET,   _______,
+    _______, _______, _______, _______, _______, _______, RGB_HUI, RGB_SAI, RGB_VAI, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, RGB_HUD, RGB_SAD, RGB_VAD, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+)
+
+};
+// clang-format on
+
+// let us assume we start with both layers off
+bool toggle_lwr = false;
+bool toggle_rse = false;
+
+static inline void led_lwr(const bool on) {
+#ifdef LED_NUM_LOCK_PIN
+    writePin(LED_NUM_LOCK_PIN, on);
+#endif
+}
+
+static inline void led_rse(const bool on) {
+#ifdef LED_SCROLL_LOCK_PIN
+    writePin(LED_SCROLL_LOCK_PIN, on);
+#endif
+}
+
+static inline void led_caps(const bool on) {
+#ifdef LED_CAPS_LOCK_PIN
+    writePin(LED_CAPS_LOCK_PIN, !on);
+#endif
+}
+
+bool led_update_user(led_t led_state) {
+    // Disable the default LED update code, so that lock LEDs could be reused to show layer status.
+    return false;
+}
+
+void matrix_scan_user(void) {
+    led_lwr(toggle_lwr);
+    led_rse(toggle_rse);
+    led_t led_state = host_keyboard_led_state();
+    led_caps(led_state.caps_lock);
+    if (layer_state_is(_ADJ)) {
+        led_lwr(true);
+        led_rse(true);
+    }
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t* record) {
+    switch (keycode) {
+        case (TT(_LWR)):
+            if (!record->event.pressed && record->tap.count == TAPPING_TOGGLE) {
+                // This runs before the TT() handler toggles the layer state, so the current layer state is the opposite of the final one after toggle.
+                toggle_lwr = !layer_state_is(_LWR);
+            }
+            return true;
+            break;
+        case (TT(_RSE)):
+            if (record->event.pressed && record->tap.count == TAPPING_TOGGLE) {
+                toggle_rse = !layer_state_is(_RSE);
+            }
+            return true;
+            break;
+        default:
+            return true;
+    }
+}
+
+#ifdef ENCODER_ENABLE
+
+#define MEDIA_KEY_DELAY 10
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    uint16_t held_keycode_timer = timer_read();
+
+    if (index == 0) { /* First encoder */
+        if (clockwise) {
+            register_code(KC_VOLD);
+            while (timer_elapsed(held_keycode_timer) < MEDIA_KEY_DELAY) {
+                // no-op
+            }
+            unregister_code(KC_VOLD);
```

Comment:
This entire block can be replaced with simply: 
```suggestion
            tap_code(KC_VOLD, MEDIA_KEY_DELAY);
```